### PR TITLE
[android] update Android App build steps

### LINKED ISSUE #3333: [android] Android App README doc is not working
#### Problem
The build steps in Android App [README](https://github.com/project-chip/connectedhomeip/blob/master/src/android/CHIPTool/README.md) doc are not working.

 #### Proposed Solution
Update the Android App README doc.

### PR DESCRIPTION
**Pull request by wgtdkp**
#### Problem
The build steps described in the Android App [README](https://github.com/project-chip/connectedhomeip/blob/master/src/android/CHIPTool/README.md) doc is not working.

 #### Summary of Changes
This PR makes changes:
1. Update the Android App document to first build `ot-commissioner` before trying `gradle sync`.
2. Add scripts to install `SWIG` for `ot-commissioner`;
3. Remove the bootstrapping script in `build-android-libs.sh` file since most dependencies are satisfied by CHIP and there are additional dependencies that are not available on some Linux distributions.

 Fixes #3333.

### CODE REVIEW

**Review by andy31415 — APPROVED** [review]

File: src/android/CHIPTool/README.md [hunk 1]
```diff
@@ -48,9 +48,19 @@ rsync -a out/android_arm64/lib/*.jar src/android/CHIPTool/app/libs
...
+    sudo apt-get install -y swig # "brew install swig" for macOS.
+
+    git submodule update --init --recursive third_party/ot-commissioner/repo
+    ABI=arm64-v8a API=21 ./third_party/ot-commissioner/build-android-libs.sh
+    ## JAR and .so libraries will be copy to target directories.
+    ```
+
+6. 'Gradle sync' the Android project and run.
+
+7. You will also need the "libc++\_shared.so" file in the jniLibs folder. This
```
Comment (mspang) [inline src/android/CHIPTool/README.md:63]: This goes before 6 too right?

**Review by mspang — APPROVED** [review]

**Review by rwalker-apple — APPROVED** [review]

**Comment by rwalker-apple** [comment]
@saurabhst @BroderickCarlin @hawk248 @jelderton ?

**Review by saurabhst — APPROVED** [review]

**Comment by rwalker-apple** [comment]
@wgtdkp this can land anytime you're ready.  perhaps mark this as a draft until you're done.

**Comment by wgtdkp** [comment]
> @wgtdkp this can land anytime you're ready. perhaps mark this as a draft if you're still hacking

@rwalker-apple I think this PR is ready to be merged.

**Pull request merged**